src/modules/UI/components/WalletListModal/WalletListModal.ui.js

```diff
@@ -20,8 +20,8 @@ export default class WalletListModal extends Component<Props> {
     const top = this.props.topDisplacement ? this.props.topDisplacement : 38
```

Comment:
This PR is for the FlipInput. What changes were needed in this file for the FlipInput animation?